run_brer/plugin_configs.py

```diff
@@ -189,6 +191,8 @@ def build_plugin(self):
         WorkElement = _get_workelement()
         if self.get_missing_keys():
             raise KeyError('Must define {}'.format(self.get_missing_keys()))
+        if self.get('alpha') == 0.0:
+            raise RuntimeError('Alpha value is constrained to 0.0.')
```

Comment:
I think this should be the same exception and message as above. Does the ValueError at line 154 work for you?
```suggestion
            raise ValueError('Read a non-sensical alpha value: 0.0')
```